x-pack/plugins/security_solution/public/management/links.ts

```diff
@@ -143,6 +144,16 @@ export const links: LinkItem = {
         }),
       ],
     },
+    {
+      id: SecurityPageName.sharedExceptionListDetails,
+      title: SHARED_LIST_DETAILS,
+      description: i18n.translate('xpack.securitySolution.appLinks.sharedListDetailsDescription', {
+        defaultMessage: 'Manage shared exception list details',
+      }),
+      landingIcon: IconExceptionLists,
+      path: '',
```

Comment:
Why is this `path` empty?

Reply:
Because it is already defined here https://github.com/elastic/kibana/pull/145605/files#diff-26a9ae1e6beaf84f8ba83da136acba3e62f6e99208bb6f9b52fd2adf19b3effdR49 
and if passed the `path` here it will append it to the URL

Please let me know, if this is not the correct way 🙏🏻

Reply:
I think there is no need to add anything to this `links.ts` file - the items in this array should be the ones that appear on the Security / Management page, but we don't want to display the _Shared list details_ there. @dasansol92 what do you think?

Reply:
I agree I don't see that we need to add an entry for the Shared Page details in the `links.ts`. 

However, I had to add a new entry to it because wanted to fix the `Page Title` in the `browser tab` being empty, and it seems this file is where to get the page name from.

Reply:
In this case, I think it's better to add it as a link of the existing `EXCEPTIONS` page:

```
[...]
{
      id: SecurityPageName.exceptions,
      title: EXCEPTIONS,
      description: i18n.translate('xpack.securitySolution.appLinks.exceptionsDescription', {
        defaultMessage: 'Create and manage exceptions to prevent the creation of unwanted alerts.',
      }),
      landingIcon: IconExceptionLists,
      path: EXCEPTIONS_PATH,
      skipUrlState: true,
      hideTimeline: true,
      globalSearchKeywords: [
        i18n.translate('xpack.securitySolution.appLinks.exceptions', {
          defaultMessage: 'Exception lists',
        }),
      ],
      links: [
        {
          id: SecurityPageName.rulesCreate,
          title: SHARED_LISTS_TITLE,
          path: SHARED_LISTS_PATH,
          skipUrlState: true,
          hideTimeline: true,
        },
      ]
    },
[...]
```

Reply:
Thanks 🙏🏻, it works https://github.com/elastic/kibana/pull/145605/commits/bb05e2612c223d5ce399bea8ee674bffcf6351ed